content/shadow-dom/sections/removing-shadow-dom-styles.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Removing Shadow DOM styles
+---
+
+### Removing Shadow DOM styles
+
+In Atom `1.13` the Shadow DOM got removed from text editors. For more background on the reasoning, check out the [Pull Request](https://github.com/atom/atom/pull/12903) where it was removed. In this guide you will learn how to migrate your theme or package.
+
+
+#### Summary
+
+Here a quick summary to see all the changes at a glance:
+
+Before | +/- | After
+---|---|---
+`atom-text-editor::shadow {}` | - `::shadow` | `atom-text-editor {}`
+`.class /deep/ .class {}` | - `/deep/` | `.class .class {}`
+`index.atom-text-editor.less` | - `.atom-text-editor` | `index.less`
```

Comment:
It isn't clear that this is a file name whereas all the other examples are CSS selectors. Perhaps remove this line and leave it for the context-specific stylesheets section below?